docs/fundamentals/syslib-diagnostics/source-generator-overview.md

```diff
@@ -168,6 +168,7 @@ The following table provides an index to the `SYSLIB1XXX` diagnostics in .NET 6
 | [SYSLIB1228][1228] | (Reserved for System.Text.Json.SourceGeneration.) |
 | [SYSLIB1229][1229] | (Reserved for System.Text.Json.SourceGeneration.) |
 | [SYSLIB1230][1230] | Deriving from a `GeneratedComInterface`-attributed interface defined in another assembly is not supported. |
+| [SYSLIB5003][5003] | SVE is a preview feature can be used by enabling `EnablePreviewFeatures` flag. |
```

Comment:
We should add this link onto [experimental-overview.md](https://github.com/dotnet/docs/blob/main/docs/fundamentals/syslib-diagnostics/experimental-overview.md) instead of this page. This page is specifically for source-generator diagnostics.

I already have this item included in the table on that page; it just needs to become a link.